src/projects/projects-router.js

```diff
@@ -58,6 +58,27 @@ projectsRouter
       }
     }
 
+    // validate project name
+    const nameError = ProjectsService.validateName(name);
+    if (nameError)
+      return res
+        .status(400)
+        .json({ error: `Project name ${name} ${nameError}` });
+    // validate description
+    const descError = ProjectsService.validateProjectDescription(description);
+    if (descError)
+      return res
+        .status(400)
+        .json({ error: `Description "${description}" ${descError}` });
```

Comment:
Since the description can be quite long, we don't want to send back the entire description as part of the error. Let's just use the word description. 

```suggestion
        .json({ error: `Description  ${descError}` });
```